index.html

```diff
@@ -14,7 +14,7 @@
   <header>
     <div id="welcome">Welcome</div>
```

Comment:
- [ ] Although your page looks great 👍 , you need to enclose the word "Welcome" in an HTML link tag as it is a requirement. The link tag is a fundamental component of HTML and is used to create hyperlinks that allow users to jump from one webpage to another, or to a specific location within a webpage

Reply:
I need help fixing horizontal scrolling